code/modules/clothing/masks/monitor.dm

```diff
@@ -9,29 +9,45 @@
 
 	icon = 'icons/mob/monitor_icons.dmi'
 	icon_override = 'icons/mob/monitor_icons.dmi'
-	icon_state = "ipc_blank"
-	item_state = null
+	icon_state = "monitor"
 
 	var/monitor_state_index = "blank"
-	var/global/list/monitor_states = list(
-		"blank" =    "ipc_blank",
-		"pink" =     "ipc_pink",
-		"red" =      "ipc_red",
-		"green" =    "ipc_green",
-		"blue" =     "ipc_blue",
-		"breakout" = "ipc_breakout",
-		"eight" =    "ipc_eight",
-		"goggles" =  "ipc_goggles",
-		"heart" =    "ipc_heart",
-		"monoeye" =  "ipc_monoeye",
-		"nature" =   "ipc_nature",
-		"orange" =   "ipc_orange",
-		"purple" =   "ipc_purple",
-		"shower" =   "ipc_shower",
-		"static" =   "ipc_static",
-		"yellow" =   "ipc_yellow"
+
+	var/list/standard_monitors = list(
```

Comment:
The ````global```` was there to ensure that this list is not represented multiple times in memory and should probably be put back.